### ISSUE TITLE
images in stack and kube spec broken

### ISSUE THREAD

**Issue by BillMills** [dockersamples/example-voting-app#177]
TLDR: need to rebuild and repush `dockersamples/examplevotingapp_vote:before`, `dockersamples/examplevotingapp_result:before` and / or `dockersamples/examplevotingapp_worker`

Steps to repro:

 - clone this repo
 - `docker stack deploy -c docker-stack.yml demo`
 - go to `localhost:5000` and vote, works great
 - go to `localhost:5001`: **votes not reported correctly**
 - logs:

```
docker service logs <results service>

...
demo_result.1.68d5fvfopn07@docker-desktop    | Waiting for db
demo_result.1.68d5fvfopn07@docker-desktop    | Waiting for db
demo_result.1.68d5fvfopn07@docker-desktop    | Waiting for db
...

docker service logs <database service>

demo_db.1.nx2lcrcpmswa@docker-desktop    | FATAL:  password authentication failed for user "postgres"
demo_db.1.nx2lcrcpmswa@docker-desktop    | DETAIL:  Connection matched pg_hba.conf line 95: "host all all all md5"
```

This is not too surprising, since:

```
docker container exec -it <results container> bash
root@814876b4a55a:/app# cat server.js 

...
async.retry(
  {times: 1000, interval: 1000},
  function(callback) {
    pg.connect('postgres://postgres@db/postgres', function(err, client, done) {
...
```

Should be `postgres://postgres:postgres@db/postgres`. Hacking that in fixes the password problem but then makes database complaints that there's no 'votes' relation, which is where I gave up on this particular rabbit hole.

Meanwhile, `docker-compose up` works fine for me, presumably because `docker-compose.yml` builds everything from source.

**Comment by BretFisher** [comment]
We're sorry for the late response. Linux images have been refreshed, cleaned up, and are now multi-platform. If you're still having issues with the latest images, feel free to reopen this ticket.
